net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -346,6 +592,71 @@ private static void doInitialization() throws CheckstylePluginException {
         CheckstyleLog.log(e, "Could not read metadata " + metadataFile); //$NON-NLS-1$
       }
     }
+
+    fillWithRemainingMetadata();
+  }
+
+  /**
+   * Create repository of Module Details from checkstyle metadata and sevntu checks metadata.
```

Comment:
remove "sevntu" word completely from source code, "sevntu" is just one from many extensions, ti should not be special at all.

Reply:
Done